src/conductor/client/interfaces/metadata_client_interface.py

```diff
@@ -0,0 +1,45 @@
+from abc import ABC, abstractmethod
+from typing import Optional, List
+from conductor.client.http.models.workflow_def import WorkflowDef
+from conductor.client.http.models.task_def import TaskDef
+
+class MetadataClientInterface(ABC):
```

Comment:
shouldn't this be just MetadataClient?

Reply:
Yes, we can do that but I was trying to keep it more _**Pythonic**_ - https://realpython.com/python-interface/#using-abcabcmeta.

Reply:
Renamed all interfaces